cli/config/compose/profiles/fleet/docker-compose.yml

```diff
@@ -16,7 +16,7 @@ services:
       - xpack.security.authc.api_key.enabled=true
       - ELASTIC_USERNAME=elastic
       - ELASTIC_PASSWORD=changeme
-    image: "docker.elastic.co/observability-ci/elasticsearch:${stackVersion:-8.0.0-SNAPSHOT}"
+    image: "docker.elastic.co/elasticsearch/elasticsearch:${stackVersion:-8.0.0-SNAPSHOT}"
```

Comment:
We're replacing the namespace because the `observability-ci` could not be up-to-date with latest SNAPSHOT, which is version we want to test in master. Consuming the upstream will allow us to discover potential bugs the soonest